tools/mpy_cross_all.py

```diff
@@ -6,6 +6,7 @@
 argparser = argparse.ArgumentParser(description="Compile all .py files to .mpy recursively")
 argparser.add_argument("-o", "--out", help="output directory (default: input dir)")
 argparser.add_argument("--target", help="select MicroPython target config")
+argparser.add_argument("--mpy_cross", help="full path/name of the mpy-cross compiler: $(MPY-CROSS) in mkenv.mk")
```

Comment:
The option name should be "--mpy-cross". The help should be "full path/name of the mpy-cross tool" (extra adds confusion).